gpkit/constraints/geometric_program.py

```diff
@@ -44,35 +46,28 @@ class GeometricProgram(NomialData):
     """
 
     def __init__(self, cost, constraints, substitutions=None, verbosity=1):
-        # pylint: disable=too-many-locals,super-init-not-called
-        # note: any substitutions in constraints are ignored -- overwritten
-        # by substitutions kwarg. This will be fixed when GeometricProgram
-        # inherits from ConstraintSet.
-        self.cost = cost
-        self.constraints = constraints
-        self.substitutions = substitutions if substitutions else {}
-
-        ## Create list of substituted posynomials <= 1
+        # pylint:disable=super-init-not-called
```

Comment:
maybe this pylint disable should go down by the `CostedConstraintSet` init so it's more obvious which init is not being called